### ISSUE TITLE
[BUG] Bounds

### ISSUE THREAD

**Issue by EquipableMC** [ShaneBeee/SkBee#603]
### What happened?

On bound enter: does not fire even if it is correct.

### Steps to reproduce

1. create a bound
2. add a broadcast
3. reload skript

### Expected behavior

It should broadcast whatever you're broadcasting, but it doesnt, even if you check the ID of the bound.

### SkBee Version

3.4.2

### Server Version

Paper 1.19.4

### SkBee Info

skbee info
[12:54:05 INFO]: --- [SkBee Loading Info] ---
[12:54:05 INFO]: - [NBTAPI] Found Spigot: v1_19_R3! Trying to find NMS support
[12:54:05 INFO]: - [NBTAPI] NMS support 'MC1_19_R3' loaded!
[12:54:05 INFO]: - NBT Elements successfully loaded
[12:54:05 INFO]: - Text Component Elements successfully loaded
[12:54:05 INFO]: - Advancement Elements successfully loaded
[12:54:05 INFO]: - BossBar Elements successfully loaded
[12:54:05 INFO]: - Bound Elements successfully loaded
[12:54:05 INFO]: - Damage Source elements disabled (Requires Minecraft 1.20.4+)
[12:54:05 INFO]: - Display Entity elements successfully loaded
[12:54:05 INFO]: - Fishing elements successfully loaded
[12:54:05 INFO]: - Game Event Elements successfully loaded
[12:54:05 INFO]: - Particle Elements successfully loaded
[12:54:05 INFO]: - RayTrace elements successfully loaded
[12:54:05 INFO]: - Recipe Elements successfully loaded
[12:54:05 INFO]: - Scoreboard Elements successfully loaded
[12:54:05 INFO]: - Scoreboard Objective Elements successfully loaded
[12:54:05 INFO]: - Statistic Elements successfully loaded
[12:54:05 INFO]: - Structure Elements successfully loaded
[12:54:05 INFO]: - Minecraft Tag elements successfully loaded
[12:54:05 INFO]: - Team Elements successfully loaded
[12:54:05 INFO]: - Tick Manager elements disabled (Requires Minecraft 1.20.4+)
[12:54:05 INFO]: - Villager Elements successfully loaded
[12:54:05 INFO]: - Virtual Furnace Elements disabled via config
[12:54:05 INFO]: - World Border Elements successfully loaded
[12:54:05 INFO]: - World Creator Elements successfully loaded
[12:54:05 INFO]: - Plugin is up to date!
[12:54:05 INFO]: --- [Server Info] ---
[12:54:05 INFO]: Server Version: git-Paper-550 (MC: 1.19.4)
[12:54:05 INFO]: Skript Version: 2.8.3
[12:54:05 INFO]: Skript Addons:
[12:54:05 INFO]: - skript-placeholders v1.6.0
[12:54:05 INFO]: - skript-gui v1.3
[12:54:05 INFO]: - DiSky v4.12.2-beta1
[12:54:05 INFO]: - skript-reflect v2.4
[12:54:05 INFO]: - skript-citizens v1.0.0
[12:54:05 INFO]: - SkReloader v1.0
[12:54:05 INFO]: - SkLogs v1.2.1
[12:54:05 INFO]: - Skript-Packet v2.2.0
[12:54:05 INFO]: SkBee Version: 3.4.2
[12:54:05 INFO]: SkBee Website: https://github.com/ShaneBeee/SkBee

**Comment by ShaneBeee** [comment]
Using the following code:
```vb
on bound enter:
	broadcast "&aENTER: &7%bound id of event-bound%"

on bound exit:
	broadcast "&cEXIT: &7%bound id of event-bound%"
```
I cannot replicate your issue.
<img width="966" alt="Screenshot 2024-03-27 at 9 58 50 AM" src="https://github.com/ShaneBeee/SkBee/assets/20008482/c4ae6aca-99f9-4c5f-91b4-55029b28007f">

**Comment by EquipableMC** [comment]
I can send a video shortly

**Comment by ShaneBeee** [comment]
Please provide your SkBee config

**Comment by EquipableMC** [comment]
Sure thing, 1 second :)

**Comment by EquipableMC** [comment]
[config.txt](https://github.com/ShaneBeee/SkBee/files/14776678/config.txt)

**Comment by ShaneBeee** [comment]
hmm everything looks in order

**Comment by EquipableMC** [comment]
and if I try to update the file, it just deletes my changes

**Comment by ShaneBeee** [comment]
That is odd... I did the same test in the nether and it worked fine.
<img width="966" alt="Screenshot 2024-03-27 at 10 06 25 AM" src="https://github.com/ShaneBeee/SkBee/assets/20008482/4167c8df-18b6-4176-a8cb-b4692b46de54">

**Comment by ShaneBeee** [comment]
> and if I try to update the file, it just deletes my changes

you can't update the file while the server is running.
After the initial load when the server starts, it never reads from that file.

**Comment by EquipableMC** [comment]
`import:
	org.bukkit.Bukkit
	
command //skbee_bound <string>:
	permission: *
	trigger:
		set {_worldEdit} to Bukkit.getServer().getPluginManager().getPlugin("FastAsyncWorldEdit")
		set {_session} to {_worldEdit}.getSession(player)
		set {_selection} to {_session}.getSelection()
		
		if {_selection} is set:
			set {_min} to {_selection}.getMinimumPoint()
			set {_max} to {_selection}.getMaximumPoint()
			set {_world} to {_selection}.getWorld()
			
			set {_minLoc} to location({_min}.getX(), {_min}.getY(), {_min}.getZ(), {_world})
			set {_maxLoc} to location({_max}.getX(), {_max}.getY(), {_max}.getZ(), {_world})
			
			create bound with id arg between {_minLoc} and {_maxLoc}
			send "&dCreated bound ""%arg%"" between %{_min}% and %{_max}%!"
		else:
			send "&cYou must make a selection first."`

**Comment by EquipableMC** [comment]
thats the code I used to create the bound

**Comment by EquipableMC** [comment]
> > and if I try to update the file, it just deletes my changes
> 
> you can't update the file while the server is running. After the initial load when the server starts, it never reads from that file.

ahh gotcha

**Comment by ShaneBeee** [comment]
yeah, my guess is FAWE isn't returning the right values for you, and there's not much I can do about that.
(EX: FAWE may not be including the world in the location object)

**Comment by EquipableMC** [comment]
ahh gotcha, alright thank you shane have a good day
